assets/js/breadcrumbs/mobile-breadcrumb.js

```diff
@@ -0,0 +1,129 @@
+var config = {
+  containerString: 'va-breadcrumbs',
+  listString: 'va-breadcrumbs-list',
+  jsHiddenClass: 'js-hide',
+  jsVisualClass: 'js-visual',
+  mobileClass: 'va-nav-breadcrumbs-list__mobile-link',
+  triggerDelay: 500,
+  triggerWidth: 425,
+};
```

Comment:
I wonder if we could have `triggerWidth` be derived from what CSS has set as the small device width. Probably just wishful thinking.

Reply:
@ncksllvn I like this suggestion a lot. I've been looking through the CSS and found 481px was the small-screen SCSS variable. I'm going to make that my breakpoint width for these breadcrumbs for consistency.